src/lib/big.ts

```diff
@@ -13,6 +13,22 @@ declare module 'big.js' {
     ceil(): Big
     roundHalfUp(decimalPlaces?: number): Big
     floor(decimalPlaces?: number): Big
+    /**
+     * Returns a string representing the value in normal notation to a fixed number of decimal places,
+     * without trailing zeros.
+     * @param dp Decimal places (0 to 1e+6 inclusive)
+     * @param rm Rounding mode (0 to 3)
+     *            0 = Rounds towards zero (truncate)
+     *            1 = Rounds towards nearest neighbor, ties toward zero
+     *            2 = Rounds towards nearest neighbor, ties away from zero
+     *            3 = Rounds away from zero
+     * @returns String representation without trailing zeros
+     * @throws If dp or rm is invalid
+     */
+    toFixedNoTrailing(
```

Comment:
can you add some tests of this to show it in action?

Reply:
Good catch! Added